csharp-selenium-webdriver-sample/SampleTest.cs

```diff
@@ -6,57 +6,74 @@
 using System;
 using System.IO;
 using Selenium.Axe;
-using OpenQA.Selenium.Chrome;
 
 namespace CSharpSeleniumWebdriverSample.Test
 {
     [TestClass]
-    [DeploymentItem("SamplePage.html")]
-    [DeploymentItem("chromedriver.exe")]
-    [DeploymentItem("geckodriver.exe")]
+   
     public class SampleTest
     {
+        // Instance to control the browser.
+        // The OpenQA.Selenium.IWebDriver interface is the main interface to use for testing, which represents an idealized web browser.
         private IWebDriver _webDriver;
 
         [TestCleanup]
         public virtual void TearDown()
         {
+            // You should quit the driver on TearDown which will close every associated window.
             _webDriver?.Quit();
         }
 
 
         [TestMethod]
+        // First DataRow Attribute will create a new instance of the test method and pass Chrome as a paramter, so it'll test on ChromDriver
+        // Second DataRow Attribute will create a new instance of the test method and pass Firefox as a paramter, so it'll test on FirefoxDriver
+        // Instead of useing DataRow Attribute, you can create a helper fucntion that takes the the broswer type as a paremter
```

Comment:
spelling: fucntion -> function

Reply:
There a few other spelling errors I missed in my first review that the VS Code "Code Spell Checker" extension caught; definitely recommend installing that and using it to check over the rest of the comments :)